src/Microsoft.MobileBlazorBindings/Elements/Picker.cs

```diff
@@ -0,0 +1,142 @@
+        /// </summary>
+        [Parameter] public string FontFamily { get; set; }
+        /// <summary>
+        /// Gets the size of the font for the label.
+        /// </summary>
+        [Parameter] public double? FontSize { get; set; }
+        /// <summary>
+        /// Gets or sets the horizontal alignment of the Text property. This is a bindable property.
+        /// </summary>
+        [Parameter] public XF.TextAlignment? HorizontalTextAlignment { get; set; }
+        /// <summary>
+        /// Gets or sets the <see cref="T:Xamarin.Forms.Color" /> for the text of this Label. This is a bindable property.
+        /// </summary>
+        /// <value>
+        /// The <see cref="T:Xamarin.Forms.Color" /> value.
+        /// </value>
+        [Parameter] public XF.Color? TextColor { get; set; }
+        /// <summary>
+        /// Gets or sets the <see cref="T:Xamarin.Forms.Color" /> for the tite of this Label. This is a bindable property.
+        /// </summary>
+        /// <value>
+        /// The <see cref="T:Xamarin.Forms.Color" /> value.
+        /// </value>
+        [Parameter] public XF.Color? TitleColor { get; set; }
+        [Parameter] public XF.TextTransform? TextTransform { get; set; }
+        /// <summary>
+        /// Gets or sets the vertical alignement of the Text property. This is a bindable property.
+        /// </summary>
+        [Parameter] public XF.TextAlignment? VerticalTextAlignment { get; set; }
+
+        static Picker()
+        {
+            ElementHandlerRegistry.RegisterElementHandler<Picker<T>>(renderer => new PickerHandler(renderer, new XF.Picker()));
+        }
+
+        protected override void RenderAttributes(AttributesBuilder builder)
+        {
+            base.RenderAttributes(builder);
+
+            if (CharacterSpacing != null)
+            {
+                builder.AddAttribute(nameof(CharacterSpacing), AttributeHelper.DoubleToString(CharacterSpacing.Value));
+            }
+            if (FontAttributes != null)
+            {
+                builder.AddAttribute(nameof(FontAttributes), (int)FontAttributes.Value);
+            }
+            if (FontFamily != null)
+            {
+                builder.AddAttribute(nameof(FontFamily), FontFamily);
+            }
+            if (FontSize != null)
+            {
+                builder.AddAttribute(nameof(FontSize), AttributeHelper.DoubleToString(FontSize.Value));
+            }
+            if (HorizontalTextAlignment != null)
+            {
+                builder.AddAttribute(nameof(HorizontalTextAlignment), (int)HorizontalTextAlignment.Value);
+            }
+            if (ItemDisplayBinding != null)
+            {
+                builder.AddAttribute(nameof(ItemDisplayBinding), ItemDisplayBinding);
+            }
+            if (ItemsSource != null)
+            {
+                builder.AddAttribute(nameof(ItemsSource), AttributeHelper.ObjectToDelegate(ItemsSource));
+            }
+            if (SelectedIndex != -1)
```

Comment:
I think it would make sense to remove this `if` and Add attribute unconditionally.
That would make possible to clear selection by setting SelectedIndex to -1.
(Currently it is set to 0 if I provide -1, which doesn't sound right)

Reply:
Good catch, if check removed.